.travis.yml

```diff
@@ -9,9 +9,9 @@ cache:
 
 matrix:
   include:
-    - env: PYTHON_VERSION="3.5" NUMPY_VERSION="1.13.1" SCIPY_VERSION="0.19.1"
+    - env: PYTHON_VERSION="3.5" NUMPY_VERSION="*" SCIPY_VERSION="*"
```

Comment:
The goal here is to test a variety of version combinations, which would not be the case if we only install latest numpy/scipy.